api/src/main/resources/default-configuration.properties

```diff
@@ -47,6 +47,14 @@ any23.extraction.metadata.domain.per.entity=off
 # registered the RDFa 1.0 legacy one (org.deri.any23.extractor.rdfa.RDFaExtractor).
```

Comment:
There should be no mention of ```org.deri.any23``` in this file. It should be ```org.apache.any23```, can you please correct this. Thanks